pkg/manifests/manifests.go

```diff
@@ -1710,6 +1710,13 @@ func (f *Factory) PrometheusK8s(grpcTLS *v1.Secret, trustedCABundleCM *v1.Config
 
 	for i, container := range p.Spec.Containers {
 		switch container.Name {
+		case "prometheus":
+			// TODO (JoaoBraveCoding): Once prometheus-operator adds CRD support to configure startupProbe directly
+			// we should use that instead of using strategic merge patch
```

Comment:
I tend to link the bug in the comment as it's more obvious than going through the Git log :)

```suggestion
			// Increase the startup probe timeout to 1h from 15m to avoid restart failures when the WAL replay takes a long time.
			// See https://issues.redhat.com/browse/OCPBUGS-4168 for details.
			// TODO (JoaoBraveCoding): Once prometheus-operator adds CRD support to configure startupProbe directly
			// we should use that instead of using strategic merge patch
```